certification/internal/policy/operator/deployable_by_olm_test.go

```diff
@@ -155,7 +92,7 @@ var _ = Describe("DeployableByOLMCheck", func() {
 		})
 		Context("When index image is in a private registry and CSV has been created successfully", func() {
 			BeforeEach(func() {
-				deployableByOLMCheck.dockerConfig = filepath.Join(tmpDockerDir, registryConfigDir, registryConfigFilename)
+				deployableByOLMCheck.dockerConfig = "./testdata/dockerconfig.json"
```

Comment:
Can we comment that this file is intended to be empty? Seeing that empty file in the test data confused me till I came to this code.